### PR TITLE
feat: Create milky and exhaust enchants while also refractoring to re…

### PR DESCRIPTION
**Pull request by CollinHerber**
…usable code

### CODE REVIEW

**Comment by Articdive** [comment]
A ton of the copyrights are DHExample module and not DHEnchants
That might also be my mistake but great if we can fix it.

File: src/main/java/com/dreamhorizon/enchantments/objects/PotionEffectType.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public enum BUFFS {
+        ABSORPTION,
+        DAMAGE_RESISTANCE,
+        FIRE_RESISTANCE,
+        INCREASE_DAMAGE,
+        INVISIBILITY,
+        JUMP,
+        REGENERATION,
+        NIGHT_VISION,
+        SLOW_FALLING,
```
Comment (Articdive) [inline src/main/java/com/dreamhorizon/enchantments/objects/PotionEffectType.java]: Dunno if Slow_FALLING (I'm presuming this is LEVITATION) is a Buff.

File: src/main/java/com/dreamhorizon/enchantments/utils/NumberUtil.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/*
```
Comment (Articdive) [inline src/main/java/com/dreamhorizon/enchantments/utils/NumberUtil.java]: Rename package to "util" so it's like core.

**Review by Articdive — CHANGES_REQUESTED** [review]

File: src/main/java/com/dreamhorizon/enchantments/objects/PotionEffectType.java (hunk 1 above)
Comment (CollinHerber) [reply src/main/java/com/dreamhorizon/enchantments/objects/PotionEffectType.java]: Moved to debuffs

File: src/main/java/com/dreamhorizon/enchantments/utils/NumberUtil.java (hunk 2 above)
Comment (CollinHerber) [reply src/main/java/com/dreamhorizon/enchantments/utils/NumberUtil.java]: Renamed

**Comment by CollinHerber** [comment]
@Articdive Ready for re-review

**Review by Articdive — APPROVED** [review]

**Pull request merged**